.travis.yml

```diff
@@ -5,6 +5,9 @@ notifications:
     on_success: change
     on_failure: always
 
+before_install:
+  - sed -i.bak -e 's|https://nexus.codehaus.org/snapshots/|https://oss.sonatype.org/content/repositories/codehaus-snapshots/|g' ~/.m2/settings.xml
```

Comment:
nexus.codehaus.org is no longer available (codehaus has been shutdown).
But this address is still used by Travis, so it needs to be replaced